### PR TITLE
Filter Landing Page cache's ObsTimeSeries

### PR DESCRIPTION
**Pull request by beets**
* Use the same logic as get_stats
* Update get_stats to use go interface instead of proto
* Update goldens

### CODE REVIEW

**Comment by beets** [comment]
Fixed tests, ptal! Hope the fix is the right one..

File: golden_response/staging/get_stats/census_pep.json [hunk 1]
```diff
@@ -332,8 +332,8 @@
       "2017": 81197,
       "2018": 83377
     },
-    "place_name": "City of Mountain View",
-    "place_dcid": "geoId/0649670",
-    "provenance_domain": "census.gov"
+    "placeName": "City of Mountain View",
```
Comment (shifucun) [inline golden_response/staging/get_stats/census_pep.json:335]: Note website also needs to update the field name and release with the mixer.

**Comment by beets** [comment]
@tjann will the change from snake case to camel case cause issues in our new notebooks? The python API simply passes through these properties, but it might be used by callers of the API.

**Comment by tjann** [comment]
No issues for notebooks

**Comment by beets** [comment]
thanks! merging..

**Pull request merged**